src/datepicker/datepicker-month-view.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, Input, TemplateRef, Output, EventEmitter} from '@angular/core';
+import {MonthViewModel, DayViewModel, DayTemplateContext} from './datepicker-view-model';
+
+@Component({
+  selector: 'tbody[ngb-datepicker-month-view]',
```

Comment:
Change to `ngbDatepickerMonthView`. Do we care that it is on `tbody`? Maybe we should relax this constraint so we leave open doors to supporting different markup?